Maxxor.PCL/ValueObject/MxDate.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using Maxxor.PCL.ValueObject.Base;
+
+namespace Maxxor.PCL.ValueObject
+{
+    public class MxDate : MxValueObject<MxDate>
+    {
+        public MxDate(int day, int month, int year)
+        {
+            if (day > 31)
```

Comment:
I would have checks here for the `day < 0` and `month < 0` cases.
You could also use `uint` since that's guaranteed to be non-zero.
The best solution would be to use [DateTime.TryParse](https://msdn.microsoft.com/en-us/library/9h21f14e(v=vs.110).aspx) since I believe that automatically takes into account invalid dates including leap years

Reply:
uint is a great idea. 

I'm avoiding DateTime specifically because it is a Date AND a Time, which is a different thing in my book.